Documentation/gettingstarted/k3s.rst

```diff
@@ -21,11 +21,12 @@ Install a Master Node
 =====================
 
 The first step is to install a K3s master node making sure to disable support
-for the default CNI plugin and the built-in network policy enforcer:
+for the default CNI plugin, the built-in network policy enforcer, and the
+built-in kube-proxy.
 
 .. code-block:: shell-session
 
-    curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC='--flannel-backend=none --disable-network-policy' sh -
+    curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC='--flannel-backend=none --disable-network-policy --disable-kube-proxy' sh -
```

Comment:
Does this then rely on Cilium's kube proxy replacement? Does that also then impose the requirements from the [kube-proxy free](https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/#kubernetes-without-kube-proxy) guide?

Reply:
yes, the assumption is user configure cilium kube proxy replacement with strict mode which seems to the trend for cilium users, maybe we should add cilium kube proxy replacement in the comment above?

Reply:
I'm not sure that we should impose KPR on users, there may be reasons they need to not use it (eg older kernels). Maybe worth just adding a note here instead that says that if you are intending to use Cilium's kube proxy replacement, also pass `--disable-kube-proxy` when configuring k3s. Then you can also cross-link the KPR doc with a `:ref:` directive.

Reply:
I pushed the changes, I cross-link the KRP doc but the KPR link has cilium version number in it, once the doc is updated, it would reference an old version link?

Reply:
I cross linked the stable URL https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free, I guess that should be fine